src/main/java/frc/robot/commands/DriveTrainCommand.java

```diff
@@ -0,0 +1,46 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import java.util.Map;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.utils.PIDFController;
+
+public class DriveTrainCommand extends CommandBase {
+
+  public DriveTrainCommand() {
+
+  }
```

Comment:
Inject `DriveTrainSubsystem` as parameter and save an instance in `DriveTrainCommand`